src/components/Cycle/ActionMenu/actions.tsx

```diff
@@ -0,0 +1,71 @@
+import { MenuItem } from '@chakra-ui/react'
+import React from 'react'
+import { useIntl } from 'react-intl'
+
+import useCadence from 'src/state/hooks/useCadence'
+
+import { Cycle } from '../types'
+
+import messages from './messages'
+
+type OptionsProperties = {
+  onViewOldCycles?: () => void
+  onCreateOKR?: (cycleID?: string) => void
+  currentCycleID?: string
+  relatedCycles?: Cycle[]
+}
+
+type AddOKROnRelatedCycleOptionProperties = {
+  cycle: Cycle
+  onCreateOKR: (cycleID?: string) => void
+}
+
+export const Actions = ({
+  onViewOldCycles,
+  onCreateOKR,
+  currentCycleID,
+  relatedCycles,
+}: OptionsProperties) => {
+  const intl = useIntl()
+
+  return (
+    <>
+      {onViewOldCycles && (
+        <MenuItem onClick={onViewOldCycles}>
+          {intl.formatMessage(messages.explorePreviousCyclesOption)}
+        </MenuItem>
+      )}
+      {onCreateOKR && Boolean(currentCycleID) && (
+        <MenuItem onClick={() => onCreateOKR(currentCycleID)}>
+          {intl.formatMessage(messages.createOKRInThisCycleOption)}
+        </MenuItem>
+      )}
+      {onCreateOKR &&
+        relatedCycles &&
+        relatedCycles.length > 0 &&
```

Comment:
⚪️ Pebble: simplificando dessa maneira o typescript vai ter o mesmo output.
```suggestion
        relatedCycles?.length > 0 &&
```